scripts/sysman/memoryTimestampValidBits.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Memory Timestamp Valid Bits"
+version: "1.8"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Memory Timestamp Valid Bits Extension Name"
+version: "1.8"
+name: $S_MEMORY_TIMESTAMP_VALID_BITS_EXP_NAME
+value: '"$XS_extension_mem_timestamp_valid_bits"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Memory Timestamp Valid Bits Extension Version(s)"
+version: "1.8"
+name: $s_mem_timestamp_bits_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for Memory bandwidth"
```

Comment:
Can we update to "Valid bits count for memory timestamp value" ?

Reply:
fixed